src/Tests/Microsoft.NET.Sdk.WorkloadManifestReader.Tests/SdkDirectoryWorkloadManifestProviderTests.cs

```diff
@@ -247,6 +247,46 @@ var sdkDirectoryWorkloadManifestProvider
                 .BeEquivalentTo("iOSContent");
         }
 
+        [Fact]
+        public void ItShouldFallbackWhenFeatureBandHasNoManifests()
+        {
+            var testDirectory = _testAssetsManager.CreateTestDirectory().Path;
+            var fakeDotnetRootDirectory = Path.Combine(testDirectory, "dotnet");
+
+            // Write 5.0.100 manifests (Expected fallback version)
+            var manifestDirectory5 = Path.Combine(fakeDotnetRootDirectory, "sdk-manifests", "5.0.100");
+            Directory.CreateDirectory(manifestDirectory5);
+            string androidManifestFileContent = "Android";
+            string iosManifestFileContent = "iOS";
+            Directory.CreateDirectory(Path.Combine(manifestDirectory5, "Android"));
+            File.WriteAllText(Path.Combine(manifestDirectory5, "Android", "WorkloadManifest.json"), androidManifestFileContent);
+            Directory.CreateDirectory(Path.Combine(manifestDirectory5, "iOS"));
+            File.WriteAllText(Path.Combine(manifestDirectory5, "iOS", "WorkloadManifest.json"), iosManifestFileContent);
+
+            // Write 4.0.100 manifests
+            var manifestDirectory4 = Path.Combine(fakeDotnetRootDirectory, "sdk-manifests", "4.0.100");
+            Directory.CreateDirectory(manifestDirectory4);
+            Directory.CreateDirectory(Path.Combine(manifestDirectory4, "Mock"));
+            File.WriteAllText(Path.Combine(manifestDirectory4, "Mock", "WorkloadManifest.json"), string.Empty);
+
+            // Write 7.0.100 manifests
+            var manifestDirectory7 = Path.Combine(fakeDotnetRootDirectory, "sdk-manifests", "7.0.100");
+            Directory.CreateDirectory(manifestDirectory7);
+            Directory.CreateDirectory(Path.Combine(manifestDirectory7, "Mock"));
+            File.WriteAllText(Path.Combine(manifestDirectory7, "Mock", "WorkloadManifest.json"), string.Empty);
+
+            var knownWorkloadsFilePath = Path.Combine(fakeDotnetRootDirectory, "sdk", "6.0.100", ".knownWorkloadIds");
+            Directory.CreateDirectory(Path.GetDirectoryName(knownWorkloadsFilePath)!);
+            File.WriteAllText(knownWorkloadsFilePath, "[\"Android\",\"iOS\"]");
+
+            var sdkDirectoryWorkloadManifestProvider
```

Comment:
I think tests for this should cover resolving to manifests that are in the current version as well as previous versions.  So I'd do something like this:

- Have iOS and Android manifests in the 7.0.100 folder
- Have an iOS manifest in the 6.0.100 folder
- Have iOS and Android manifests in the 5.0.100 folder
- Have an Android manifest in the 4.0.100 folder

Each manifest should have different contents depending on the feature band it's in so we can tell which folder it was resolved from.  The test should verify that the iOS manifest was resolved from 6.0.100 and Android was resolved from 5.0.100.